packages/strapi-plugin-upload/admin/src/components/UploadList/RowItem.js

```diff
@@ -6,7 +6,8 @@ import CardImgWrapper from '../CardImgWrapper';
 import InfiniteLoadingIndicator from '../InfiniteLoadingIndicator';
 
 const RowItem = ({
-  file,
+  // file,
```

Comment:
Do you still need this?

Reply:
You might need it when designing the card so I left it just to let you know it's already there.